### ISSUE TITLE
JAWS is not announcing "Group Color" label name when color radio buttons receiving focus.

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio#25777]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Info and Relationships; #FTP; #ScreenReader; #Win11; #Win32; #Element:Label; #JAWS;

### Environment Details:
Azure Data Studio Version: 1.48.0 (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296
Screen Reader: JAWS 2024, NVDA 2023.3.2

### Repro Steps:

1. Launch Azure Data Studio.
2. Start JAWS.
3. Tab to the Connections icon and press enter to Select (if not already selected) from left bar.
4. Tab to the icon that has a tool tip of 'New Server Group' icon and press enter.
5. Enter the name for the Server group name and a description.
6. Tab to "Group color" section radio buttons.
7. Observe JAWS is not announcing "Group Color" label name when color radio buttons receiving focus.

### Actual Result:
JAWS is not announcing "Group Color" label name when color radio buttons receiving focus.

**Note: This issue does not repro with NVDA.**

### Expected Result:
JAWS should be announcing the "Group Color" label name when color radio buttons receiving focus.

### User Impact:
Screen reader users will not be able to understand the purpose of the color radio buttons and will get confused.

### Attachment:
[NVDA_Norepro.zip](https://github.com/user-attachments/files/16374330/NVDA_Norepro.zip)
[JAWS.zip](https://github.com/user-attachments/files/16374331/JAWS.zip)
![image](https://github.com/user-attachments/assets/dd5713e0-09ff-4736-972a-5f8b390fefd1)

**Comment by Swati1700** [comment]
Primary bug: https://github.com/microsoft/azuredatastudio-mysql/issues/234

**Comment by Samhitha66** [comment]
Rev:samhithaks

**Comment by Swati1700** [comment]
Verified the issue in the environment 
ADS Version: 1.50.0 (user setup)
Issue is still reproducing. Please find the repro attachment below:
https://github.com/user-attachments/assets/663078b9-238d-4284-8453-96a678d975fa

**Comment by Raj-ghuge** [comment]
Verified the issue in the environment
ADS Version: 1.50.0 (user setup)
The issue is does not repro, hence closing the bug.
Please find the attachment for your reference.

#Closed;
#Regressed:01-14-25

https://github.com/user-attachments/assets/468df234-fc7b-4c34-9a27-b205d7b43a82

**Comment by Swati1700** [comment]
Bug is fixed hence closing the bug.
